pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -237,14 +239,17 @@ func (volumes *VolumesV3) expandVolume(volumeID string, newSize int) error {
 		NewSize: newSize,
 	}
 
-	// save initial microversion
-	mv := volumes.blockstorage.Microversion
-	volumes.blockstorage.Microversion = "3.42"
+	// Init a local thread safe copy of the Cinder ServiceClient
+	blockstorageClient, err := openstack.NewBlockStorageV3(volumes.blockstorage.ProviderClient, volumes.epOpts)
```

Comment:
Actually there is no need to do this, since a new copy of the volume client is already initialized [here](https://github.com/kubernetes/cloud-provider-openstack/blob/9774a6829cbc0c35b65824ff7fff82b936f3c51e/pkg/cloudprovider/providers/openstack/openstack_volumes.go#L371), I'll just put extra comments into the code and it should be enough.